config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
+    password character(34),
```

Comment:
I lowered the char amount to 34 (and used fixed char types), because the MD5Crypt hashes always have that length.

Reply:
This is now an existing table, it needs to be done with an alter statement. I'm really not clear on why we can shorten to 34 though, wasn't 40 the existing, can we not have passwords that length existing?

Reply:
TBH we need column comments, that should be standard as part of our table schema. Your comments @RoiEXLab  are very useful, but we need that info beyond review. So we could update this file with things like:
`password character(34) comment 'stores md5 encrypted password, length is always 34'`

Would you mind making that update @RoiEXLab ?

Reply:
The passwords are not stored in plain text, they are hashed with the md5crypt algorithm which all have the exact length of 34 (I checked with `SELECT max(char_length(password)), min(char_length(password))` before you altered the table to allow up to 60 chars), I plan to move all of the bcrypted passwords to the new row before applying this change, I'm probably going to create a list or SQL statements which modify the DB before updating the lobby.